### PR TITLE
Convert Enterprise form pages to Protocol (#7352)

### LINKED ISSUE #7352: Update /enterprise visual design [July 8]
## Description

Update the look of /enterprise to match the new brand.

Ideally in production July 8th for launch of Firefox 68 on July 9th.

fx-enterprise-1312

Details in:
https://trello.com/c/OdBsjECC/324-update-enterprise-visual-design-only-ui-dev-3

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] /enterprise landing page has updated look
- [ ] /enterprise/signup has updated look
- [ ] /enterprise/signup/thanks has updated look

#### :heavy_exclamation_mark: Risks :heavy_exclamation_mark: 
- [ ] @stephaniehobson is on PTO next week, July 8th is tight.
- [ ] signup and thanks depend on #7141 

#### Tasks
- [x] Update /enterprise page
- [x] Re-code /enterprise to remove the table
- [x] design review 
- [ ] code review 
- [ ] Update /signup and /thanks to use Protocol basic templates
- [ ] Tweak /signup and /thanks as needed
- [ ] code review

### PR DESCRIPTION
**Pull request by stephaniehobson**
## Description

- Add protocol-basic template and style sheet
- use it for the form pages
- tweak form page styles as necessary

## Issue / Bugzilla link

Depends on #7353.
Fix #7352

## Testing

Please do not look at the /enterprise landing page as part of this. I had to copy the style sheet from #7353 to avoid SCSS errors. Before publishing this PR will have to be re-based on #7353 and incorporate any review feedback from there.

### CODE REVIEW

**Comment by stephaniehobson** [comment]
Hoping to publish this on the 8th when I am back from PTO for Firefox 68 launch on the 9th.

File: bedrock/base/templates/base-protocol-basic.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  {{ css_bundle('protocol-basic') }}
+{% endblock %}
+
+{% block body_id %}{{ body_id }}{% endblock %}
+
+{% block content %}
+  {% block main_feature %}{% endblock %}
+  <div id="main-content" class="mzp-l-content{% if self.side_nav()|trim|length or self.side_extra()|trim|length %} mzp-has-sidebar mzp-l-sidebar-right{% endif %}">
+    <main class="mzp-l-main">
+        <article class="mzp-c-article">
```
Comment (alexgibson) [inline bedrock/base/templates/base-protocol-basic.html]: Nit: indentation

File: bedrock/firefox/templates/firefox/enterprise/signup.html [hunk 2]
```diff
@@ -2,10 +2,10 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
-{% extends "base-pebbles-basic.html" %}
+{% extends "base-protocol-basic.html" %}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/enterprise/signup.html]: Not a blocker, but do these pages need to inherit from `base-protocol-basic.html`? They don't seem to feature a sidebar or a newsletter, so some of the styles seem a bit redundant. Could this use the regular protocol base template instead?

File: bedrock/firefox/templates/firefox/enterprise/index.html [hunk 3]
```diff
@@ -7,7 +7,7 @@
 {% from "macros-protocol.html" import hero with context %}
 
 {% block page_title_prefix %}{% endblock %}
-{% block page_title %}{{_('Firefox Quantum for Enterprise Web Browser | Firefox')}}{% endblock %}
+{% block page_title %}{{_('Firefox Broswer for Enterprise | Firefox')}}{% endblock %}
```
Comment (devinreams) [inline bedrock/firefox/templates/firefox/enterprise/index.html]: Minor typo fix submitted here #7409

Comment (amychurchwell) [reply bedrock/firefox/templates/firefox/enterprise/index.html]: Thanks for the catch!

**Pull request merged**